src/Modules/Backend/assets/Backend/webpack/scss/components/_authentication.scss

```diff
@@ -7,7 +7,7 @@
   }
 }
 
-#authenticationIndex {
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. 
Let's focus here on the CI issue, otherwise this PR will get out of focus (again).